testcases/kernel/controllers/memcg/stress/memcg_process_stress.c

```diff
@@ -92,13 +92,18 @@ int main(int argc, char *argv[])
 		interval = 1;
 
 	/* TODO (garrcoop): add error handling. */
-	memset(&sigint_action, 0, sizeof(sigint_action));
+	/* TODO(mw): DRY it */
```

Comment:
If the code grows, the change should be reconsidered. 
I'd change the comment to "consider DRYing".